crates/store/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -182,7 +182,9 @@ macro_rules! tensor_from_ndarray {
                     .is_standard_layout()
                     .then(|| TensorData {
                         shape,
-                        buffer: TensorBuffer::$variant(value.to_owned().into_raw_vec().into()),
+                        buffer: TensorBuffer::$variant(
+                            value.to_owned().into_raw_vec_and_offset().0.into(),
```

Comment:
We shouldn't ignore the offset. If it is non-zero, we may have a problem.
We should at least return `TensorCastError::NotContiguousStdOrder` in that case.

However, ideally we should handle arrays that are not in standard layout (and/or with an offset) by calling `.iter().collect()` instead of bailing.

Reply:
Sounds good. Will give it a try.
I want to confirm my understanding that if the Array `is_standard_layout()` the offset can be safely ignored (which is also implied by you commenting on this edit and not the one further down, where a check for standard layout had been done first)

Reply:
I don't know - the docs aren't very clear about that, but I believe it is not enough.

Reply:
You're correct, I was able to validate the suspicion with the following code (which passes, offset = 4)
```
    let array = Array::from_iter(0..16).into_shape_with_order((4, 4)).unwrap();
    let sliced_array = array.slice_move(s![1.., ..]);
    assert!(sliced_array.is_standard_layout());
    let (_, offset) = sliced_array.into_raw_vec_and_offset();
    assert!(offset.unwrap() > 0);
```
I'll make sure the test cases cover this as well